docs/book/src/quick-start.md

```diff
@@ -57,6 +57,29 @@ cd $GOPATH/src/example
 kubebuilder init --domain my.domain
 ```
 
+If there is an error similar to `dial tcp x.x.x.x:443: i/o timeout`, you need to set go proxy.
```

Comment:
It shows specific to your environment. I do not think that the quick start would be an appropriate place for that.

Reply:
All right,It is more appropriate to the issue or FAQ